collections/_article/godot-xr-community-nov-2024.md

```diff
@@ -0,0 +1,117 @@
+This features some excellent interactions both from the player point of view, as well as the dog.
+
+The #2 spot went to [Constellation Coach VR](https://itch.io/jam/godot-xr-game-jam-july-2024/rate/2830514) with 3.974 points.
+This game has you find constellations in the night sky.
+By holding up a picture of a constellation, excellent use of haptic feedback allows you to find the relevant stars.
+Successfully finding the right stars lights up the constellation.
+
+The #3 spot went to [Summer Pottery VR](https://itch.io/jam/godot-xr-game-jam-july-2024/rate/2830121) with 3.938 points.
+This game lets you do some clay pottery sculpting.
+Great use of physically interacting with the clay to shape and paint your creation, this is very well executed.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/0IxvWaj8iUo?si=em-D99NacStXyIYA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+
+Again, many of the other entries were of great quality and we highly recommend checking them out.
+
+### Godot XR Game Jam #3
+
+We are still planning the 3rd game jam which will likely be held in the week of the 10th of February 2025. Stay tuned!
+
+*Should see if we can get the page up in time and link to it*
+
+## Universal Godot VR plugin
+
+Inspired by the Universal Unreal Engine VR Mod, community member TeddyBear082 has created a mod to inject VR capabilities into existing 3D games made with Godot.
+
+TeddyBear082 previously made a name for himself with the excellent VR port of the popular game Cruelty Squad and has used the knowledge gained with that and similar projects to create a more universal mod.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/Hb9BDcCZHCY?si=Zr2Jw3MV7cY00yEE" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+
+You can find the plugin [on teddybear082's GitHub](https://github.com/teddybear082/UGVR).
+
+## Body tracking and motion capture plugins
+
+Core contributor Malcolm Nixon has created various plugins for Godot that use the new body tracking feature to make tracking data available from various sources.
+
+He has support for Rokoko, Axis Studio and Movella.
+He also has a plugin for VMC (Virtual Motion Capture), which is a standard used by a number of motion capture systems.
+
+*Should insert an image of this work?*
+
+You can find the plugins [on his GitHub](https://github.com/Malcolmnixon?tab=repositories).
+
+## Meta platform plugin
+
+Community member Decacis has released a few of his games on the Meta Horizon store and implemented part of the Meta platform SDK in support of this.
```

Comment:
Could be nice to link some of Decacis's games on the Meta Horizon Store here?

Reply:
Would be nice, wonder what the best way of doing that is without it going too commercial.

Reply:
I think it'd be fine to link to the store pages directly - we link to Steam pages on most of the release notes posts in order to credit the screenshots. Personally, I think linking to a Quest store page is basically the same as linking to a Steam store page.

Here's two links so you don't have to dig them up:

- https://www.meta.com/experiences/5816419461787331/
- https://www.meta.com/experiences/5030396847007781/

Should probably ask Decasis for permission first, though?